js/index.js

```diff
@@ -1,7 +1,99 @@
-// Iteration 1: Names and Input
+//Ejercicio 1 Names & INPUT
 
+var hacker1 = "Tomy";
 
-// Iteration 2: Conditionals
+console.log(`The driver's name is ${hacker1}`) 
 
+var hacker2 = "Tomas";
 
-// Iteration 3: Loops
\ No newline at end of file
+console.log(`The navigator's name is ${hacker2}`) 
+
+//Ejercicio 2 Conditionals 
+
+const driversNameLonger = hacker1.lenght > hacker2.length
+const navigatorsNameLonger = hacker2.length > hacker1.length
+
+if (driversNameLonger){ //Ha de cumplir la condición de su variable, sino es true salta al siguiente
+  console.log(`The driver has the longest name, it has ${hacker1.length}`)
+}
+else if (navigatorsNameLonger){//Como el nombre Tomas es + largo que Tomy se imprime y no salta a la última condición
+  console.log(`It seems that the navigator hast the longest name, it has ${hacker2.length}`)
+}
+else {//se activaría si ninguna de las de arriba fuese true
+  console.log(`Wow, you both have equally long names, ${hacker2.length}`)
+}
```

Comment:
Perfecto 👍 
buena práctica guardar las expresiones en variables, para el nombre de la variables recuerda usar "is", "can" o "has" (es una convención que nos permite mejorar la legibilidad y comprensión del código). 
Por ejemplo "driverHasLongerName" o "navigatorHasLongerName"